src/plugins/intel_npu/tools/single-image-test/main.cpp

```diff
@@ -2084,6 +2145,12 @@ static int runSingleImageTest() {
             const FilesForModelInputs &inputFiles = inputFilesForOneInfer[numberOfTestCase];
             OPENVINO_ASSERT(inputFiles.size() == inputsInfo.size(), "Number of input files ", inputFiles.size(),
                             " doesn't match network configuration ", inputsInfo.size());
+            const RefFilesForModelOutputs &refFiles = refFilesForOneInfer.empty() ? RefFilesForModelOutputs{}
+                                                                                 : refFilesForOneInfer[numberOfTestCase];
```

Comment:
I still didn't get why we are getting this `refFiles` as ` refFilesForOneInfer[numberOfTestCase];` and then getting 
`refFiles[numberOfTestCase]` again inside a function `getRefBlobFilePath()`

I assume that there is the confusion regarding this "|" layer in parsing  `refFilesForOneInfer`, which brings up this addition array layer

Reply:
I have removed the "|" layer as it may be less useful in model output.
Hence now the convention becomes: `Ref1A, Ref1B; Ref2A, Ref2B`

`Ref1A, Ref1B` will constitute the first test case (test case 0), with 2 outputs (1A and 1B)
`Ref2A, Ref2B` will constitute the second test case (test case 1), with 2 outputs (2A and 2B)

I've also cleared up the confusion in the help text. (semicolon = different test case, comma = different reference file within same test case)